src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,21 @@
+.catalog {
+  display: flex;
+  flex-wrap: wrap;
+  padding: $paddingY $paddingX;
+  width: catalogWidth($cardCountInRow);
+
+  &__item {
+    margin-right: 48px;
+    margin-bottom: 46px;
+    width: $cardWigth;
```

Comment:
Why do you set the width here? It duplicates the same prop from the card.

Reply:
fixed
and mixin added